samples/stock/demo/drag-on-axis/demo.js

```diff
@@ -9,98 +9,118 @@ function zoomOnYAxis() {
             zoomOffset = 0;
         }
 
-        axis.setExtremes(center - zoomOffset, center + zoomOffset, false);
-        axis.chart.redraw(false);
+        axis.setExtremes(center - zoomOffset, center + zoomOffset, true, false);
     }
 
-    Highcharts.addEvent(Highcharts.Chart, 'load', function () {
-        const chart = this,
-            renderer = chart.renderer,
-            yAxes = chart.yAxis.filter(axis => !axis.options.isInternal);
+    Highcharts.addEvent(Highcharts.Axis, 'afterInit', function () {
+        const axis = this,
+            renderer = axis.chart.renderer;
 
-        yAxes.forEach(yAxis => {
+        if (!axis.isXAxis && !axis.options.isInternal) {
             // Create a transparent rectangle on yAxis to create a zoom area
-            yAxis.axisZoomRect = renderer.rect()
+            axis.axisZoomRect = renderer.rect()
                 .attr({
                     width: 50,
-                    height: yAxis.height,
-                    fill: 'transparent'
+                    height: 300,
+                    fill: 'transparent',
+                    zIndex: 8
                 })
+                .addClass('highcharts-no-mousewheel')
                 .css({ cursor: 'ns-resize' })
                 .add();
 
             // Enabled zooming via mousewheel scroll
-            yAxis.axisZoomRect.on('mousewheel', event => {
-                centerZoom(yAxis, event.deltaY * 0.0001);
+            axis.axisZoomRect.on('mousewheel', event => {
+                centerZoom(axis, event.deltaY * 0.0001);
             });
 
             // Enable zooming by drag on mouse click
-            yAxis.axisZoomRect.on('mousedown', () => {
-                yAxis.drag = true;
+            axis.axisZoomRect.on('mousedown', () => {
+                axis.drag = true;
             });
 
             // Zoom on yAxis while dragging the mouse
             document.addEventListener('mousemove', event => {
-                if (yAxis.drag) {
-                    centerZoom(yAxis, event.movementY * 0.001);
+                if (axis.drag) {
+                    centerZoom(axis, event.movementY * 0.001);
```

Comment:
Let's increase the speed a little, 0.001 -> 0.002 works nice for me (touchpad) - what do you think?